net.peercoin.peercoin-qt.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "net.peercoin.peercoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "peercoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "peercoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/peercoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/peercoin/peercoin/releases/download/v0.10.4ppc/peercoin-0.10.4-x86_64-linux-gnu.tar.gz",
+          "sha256": "59af6c2a3102da91d8dfc6d1bb5da1c5f21825bb878f6741518b3eeaae51c361"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 net.peercoin.peercoin-qt.desktop /app/share/applications/net.peercoin.peercoin-qt.desktop",
+        "install -Dp -m 644 peercoin.svg /app/share/icons/hicolor/scalable/apps/net.peercoin.peercoin-qt.svg",
+        "install -Dp -m 644 net.peercoin.peercoin-qt.metainfo.xml /app/share/metainfo/net.peercoin.peercoin-qt.metainfo.xml",
+        "bash install_wrapper.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "net.peercoin.peercoin-qt.desktop"
+        },
+        {
+          "type": "file",
+          "path": "net.peercoin.peercoin-qt.metainfo.xml"
+        },
+        {
+          "type": "file",
+          "path": "peercoin.svg"
+        },
+        {
+          "type": "file",
+          "url": "https://www.peercoin.net/img/wallets_splash.png",
```

Comment:
I don't see this file addressed anywhere.

Reply:
Removed.